### ISSUE TITLE
Update kerl to support newer Catalina versions

### ISSUE THREAD

**Issue by odarriba** [asdf-vm/asdf-erlang#139]
Hi!

kerl has just released a new version that includes a fix to allow compilation on newer versions of OSX Catalina, as described [here](https://github.com/kerl/kerl/pull/337/)

It's a major version bump (1.8.7 -> 2.0.0), so there are probably other tweaks apart from just changing the version number on asdf-erlang, but without that patch it is imposible to build versions released before Erlang 22.3.1

Thanks for the awesome job on this plugin ❤️

**Comment by Stratus3D** [comment]
According to the kerl changelog this is the only possible breaking change:

> Important: possible breaking change - releases are now fetched through github tags by default (#277)

I ran `asdf list-all erlang` and wrote the output to a file, then upgraded the kerl version used by this plugin to 2.0.0, and ran `asdf list-all erlang` again and wrote the output to another file. I diffed the two files and there were no differences. I think it is ok to upgrade to kerl 2.0.0.
